content/docs/conditional-rendering.md

```diff
@@ -146,17 +146,17 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ozJddz?editors=0010)
+[**Δοκιμάστε το στο CodePen**](https://codepen.io/gaearon/pen/ozJddz?editors=0010)
 
-It works because in JavaScript, `true && expression` always evaluates to `expression`, and `false && expression` always evaluates to `false`.
+Λειτουργεί επειδή στην JavaScript, το `true && expression` πάντα αποτιμάται σε `expression`, και το `false && expression` πάντα αποτιμάται σε `false`.
```

Comment:
`Αποτιμάται` ή `κάνει evaluate`? 🤔

Reply:
@gatzjames Always evaluates to = Πάντα ισούται με?

Reply:
Δύσκολη μετάφραση. Δε ξέρω αν το ισούται είναι σωστό. Θα έλεγα ας το αφήσουμε έτσι η ας χρησιμοποιήσουμε το αγγλικό.